src/Robot-Dobot_MG400/main.py

```diff
@@ -125,6 +125,11 @@ def set_GUIparameter(self):
                         "Payload x offset": 0.0,
                         "Payload y offset": 0.0,
                         # "Payload z offset": 0.0,
+                        " ": None,
+                        "Jump": None,
+                        "Use jump mode": False,
+                        "Movement height": 0.0,
+                        "  ": None,
```

Comment:
This is a separator, right? And the different amount of spaces is just to have unique dictionary keys?
I think this is very unintuitive, can't we use a more describing dictionary key, maybe with an underscore to show it's not a real parameter, e.g.
```python
{
    ...
    "_separator1": None,
    ...
    "_separator2": None,
    ...
}
```

Also it appears a bit fragile, because it depends on the order of entries in a (unordered) dictionary. But that's another story.

Reply:
yes, it is separator. However, if you give use a non-empty key it will be used a bold label to indicate a section:
![image](https://github.com/SweepMe/instrument-drivers/assets/61842286/263ba5c0-ffba-497c-9938-dc1938934924)
You are right that this is not a good solution, but it cannot be solved within this PR, but needs to be handled in SweepMe! by revising the handling of gui parameters.